From 8c295fec7c39a5888d90df1c006b977a5a461ec3 Mon Sep 17 00:00:00 2001
From: Rong Dian <b38775@freescale.com>
Date: Tue, 14 Aug 2012 14:54:49 +0800
Subject: ENGR00219898 imx6 battery: fix coulomb data for power down system

fix battery coulomb data for power down system ,define low battery
voltage for power down system is 3.42V

Signed-off-by: Rong Dian <b38775@freescale.com>
---
 drivers/power/sabresd_battery.c |  135 ++++++++++++++++++++++-----------------
 1 file changed, 76 insertions(+), 59 deletions(-)

diff --git a/drivers/power/sabresd_battery.c b/drivers/power/sabresd_battery.c
index d781544..516fbd8 100755
--- a/drivers/power/sabresd_battery.c
+++ b/drivers/power/sabresd_battery.c
@@ -73,68 +73,85 @@ static int offset_usb_charger;
 
 
 static battery_capacity chargingTable[] = {
-    {4060,  99},
-    {4035,  98},
-    {4010,  97},
-    {3980,  96},
-    {3950,  95},
-    {3920,  94},
-    {3890,  93},
-    {3860,  92},
-    {3830,  91},
-    {3780,  90},
-    {3750,  85},
-    {3690,  80},
-    {3660,  75},
-    {3630,  70},
-    {3600,  65},
-    {3580,  60},
-    {3550,  55},
-    {3500,  50},
-    {3450,  45},
-    {3400,  40},
-    {3350,  35},
-    {3300,  30},
-    {3250,  25},
-    {3200,  20},
-    {3000,  15},
-    {2900,  10},
-    {2850,  5},
-    {2800,  0},
-    {0,  0}
+	{4050,	99},
+	{4040,	98},
+	{4020,	97},
+	{4010,	96},
+	{3990,	95},
+	{3980,	94},
+	{3970,	93},
+	{3960,	92},
+	{3950,	91},
+	{3940,	90},
+	{3930,	85},
+	{3920,	81},
+	{3910,	77},
+	{3900,	73},
+	{3890,	70},
+	{3860,	65},
+	{3830,	60},
+	{3780,	55},
+	{3760,	50},
+	{3740,	45},
+	{3720,	40},
+	{3700,	35},
+	{3680,	30},
+	{3660,	25},
+	{3640,	20},
+	{3620,	17},
+	{3600,	14},
+	{3580,	13},
+	{3560,	12},
+	{3540,	11},
+	{3520,	10},
+	{3500,	9},
+	{3480,	8},
+	{3460,	7},
+	{3440,	6},
+	{3430,	5},
+	{3420,	4},
+	{3020,	0},
 };
 static battery_capacity dischargingTable[] = {
     {4050, 100},
-    {4020,  99},
-    {3950,  98},
-    {3920,  97},
-    {3890,  96},
-    {3860,  96},
-    {3830,  95},
-    {3800,  94},
-    {3760,  93},
-    {3730,  92},
-    {3700,  91},
-    {3670,  90},
-    {3630,  85},
-    {3600,  80},
-    {3570,  75},
-    {3545,  70},
-    {3515,  65},
-    {3480,  60},
-    {3445,  55},
-    {3410,  50},
-    {3375,  45},
-    {3340,  40},
-    {3300,  35},
-    {3250,  30},
-    {3200,  25},
-    {3150,  20},
-    {3090,  15},
-    {3020,  10},
-    {2820,  5},
-    {2800,  0},
-    {0,  0}
+	{4035,	99},
+	{4020,	98},
+	{4010,	97},
+	{4000,	96},
+	{3990,	96},
+	{3980,	95},
+	{3970,	92},
+	{3960,	91},
+	{3950,	90},
+	{3940,	88},
+	{3930,	86},
+	{3920,	84},
+	{3910,	82},
+	{3900,	80},
+	{3890,	74},
+	{3860,	69},
+	{3830,	64},
+	{3780,	59},
+	{3760,	54},
+	{3740,	49},
+	{3720,	44},
+	{3700,	39},
+	{3680,	34},
+	{3660,	29},
+	{3640,	24},
+	{3620,	19},
+	{3600,	14},
+	{3580,	13},
+	{3560,	12},
+	{3540,	11},
+	{3520,	10},
+	{3500,	9},
+	{3480,	8},
+	{3460,	7},
+	{3440,	6},
+	{3430,	5},
+	{3420,	4},
+	{3020,	0},
 };
 
 u32 calibrate_battery_capability_percent(struct max8903_data *data)
-- 
1.7.9.5

